bamstats.wdl

```diff
@@ -15,7 +15,7 @@ task bamstats {
 	}
 
 	runtime {
-		docker: "quay.io/collaboratory/dockstore-tool-bamstats:1.25-7"
+		docker: "ghcr.io/dockstore/dockstore-tool-bamstats:1.25-8"
```

Comment:
Please make sure this WDL has been confirmed to actually work as it is written here, because in my testing this registry doesn't work with Cromwell/the Dockstore CLI. If you've already confirmed it works, please also let me know because then that means the issues I have with ghcr.io are my own fault.

Reply:
In fact, it's worth manually confirming every workflow in this repo works as expected with the calls we're giving, if that hasn't already been done. Most of the issues I had with this repo previously were things not working as expected.

Reply:
See above

Reply:
My bad, I misinterpreted what the GH Action is doing the first time around but I think I get it now. I still stand by what I said about not throwing warnings at new users but the most important thing is that things work.